stackstate_checks_base/stackstate_checks/base/checks/base_v2.py

```diff
@@ -0,0 +1,330 @@
+import copy
```

Comment:
Maybe move the new V2 checks to `base/checks/v2/base.py`. Also, the `CheckMixin`, `Health` and `Stateful` could be in an `util/v2` module. Whoever is using the new base class doesn't need to import them.